bot/exts/moderation/modpings.py

```diff
@@ -51,10 +52,14 @@ def __init__(self, bot: Bot):
             event_loop=self.bot.loop,
         )
 
-    async def reschedule_roles(self) -> None:
-        """Reschedule moderators role re-apply times."""
+    async def fetch_guild(self) -> None:
+        """Fetch the guild."""
         await self.bot.wait_until_guild_available()
         self.guild = self.bot.get_guild(Guild.id)
+
+    async def reschedule_roles(self) -> None:
+        """Reschedule moderators role re-apply times."""
```

Comment:
Could this perhaps be reworded a bit as this doesn't make much sense to me.

Reply:
I believe this is out of scope of this pr, and should be changed in GH-2001

Reply:
Yeah, agreed. Have done left a comment on that PR :+1: